pkg/sensors/exec/exit_test.go

```diff
@@ -163,8 +163,15 @@ func TestExitZombie(t *testing.T) {
 	logWG := testPipes.ParseAndLogCmdOutput(t, nil, nil)
 	logWG.Wait()
 
-	if err := testCmd.Wait(); err != nil {
-		t.Fatalf("command failed with %s. Context error: %v", err, ctx.Err())
+	 err := testCmd.Wait(); if err != nil {
+		if exitError, ok := err.(*exec.ExitError); ok {
+			// Process exited with a non-zero exit code
+			exitCode := exitError.ExitCode()
+			fmt.Printf("Process exited with code: %d\n", exitCode)
+	}else{
+		fmt.Printf("Error waiting for process to exit: %v\n", err)
```

Comment:
But here we will still have to `Fatal`, I think.
Also, please use t.Logf (or similar) instead of fmt.Printf